test/functions_test.py

```diff
@@ -156,6 +157,28 @@
 23
 >>> params["water"]
 5000
+>>> args = parser.parse_args(["--method", 'steep-and-release', "--digits", '1'])
+>>> params = load_params(args)
+>>> params["coffee"] = calc_coffee(params)
+>>> params["water"]
+255
+>>> params["coffee"]
+15.9375
+>>> params["water_ratio"]
+16
+>>> params["coffee_ratio"]
+1
+>>> params["method"]
+'steep-and-release'
+>>> params = filter_params(params)
+>>> params["water_ratio"]
+16
+>>> params["coffee_ratio"]
+1
+>>> params["water"]
+255
+>>> params["coffee"]
+15.9
```

Comment:
Don't we want to add `params['digits']`

Reply:
fixed in https://github.com/sepandhaghighi/mycoffee/pull/25/commits/999bf5f5309fb525fba760761131711239b477da